source/assets/stylesheets/ie8_fixes.scss

```diff
@@ -31,12 +33,23 @@
 }
 
 // Place icons in the middle of its container
-.footer-social_icons__icon {
+.social_icons__icon {
   padding-top: 6px;
   height: 22px;
 }
 
 // To support full coloured div
-.footer-social-icons {
+.social-icons {
   display: block;
 }
+
+// To set a blog article width
+.blog-header__article-header,
+.blog-article__container,
+.blog-author__container,
+.blog-comments__container,
+.blog-related__container,
+.blog-related__header {
```

Comment:
> ...set width to the main blog container it will show background body colour instead of white. That's why there is a bit fuss with containers on blog post page.

This is what I was thinking of using for the main blog post layout:

```HTML
<div class="blog-article">
  <div class="blog-article__container">
    <!-- title  -->
    <!-- author date  -->
    <!-- tags  -->
    <!-- blog post  -->
    <!-- share -->
  </div>
</div>

<!-- doesn't require a wrapper container as it'll use the body background colour -->
<div class="blog-article__container">
  <!-- author details -->
  <!-- comments -->
</div>
```

```CSS
.blog-article {
  background-color: #ffffff;
}

.blog-article__container {
  box-sizing: border-box;
  margin: 0 auto;
  max-width: 960px;
  padding: 16px;
}
```